starter-project-backend/src/tests/__tests__/ratings.test.ts

```diff
@@ -0,0 +1,107 @@
+import Rating, { IRating } from "../../models/ratings"
+import * as dbHandler from '../setupdb'
+import request from 'supertest'
+import app from "../../app"
+
+beforeAll(async () => {
+    await dbHandler.connect()
+})
+
+afterEach(async () => {
+    await dbHandler.clear()
+})
+
+afterAll(async () => {
+    await dbHandler.disconnect()
+})
+
+describe('GET /api/articles/{id}/ratings', () => {
+    it('should return status code 200', async () => {
+        const userId = "1"
+        const value = 3
```

Comment:
to test get a request I think you don't need to send a value, plus if you want to have an object before testing you have to create an article object inside before each method so that you can test get request

Reply:
Fixed